test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
 
 print("my")
-print("world")
```

Comment:
The print statements "test1" and "testtttt" don't seem very informative for a test file. Explain what these printed strings are meant to verify, or use more descriptive strings. Also, consider using assertions instead of printing to confirm expected behavior.

```suggestion
print("test1 output") # Verify func returns something related to "test1"
```